kitchen.dokken.yml

```diff
@@ -76,7 +74,12 @@ platforms:
       intermediate_instructions:
         - RUN /usr/bin/apt-get update
 
-  - name: opensuse-leap
+  - name: opensuse-leap-42
```

Comment:
42 is EOL. You'll want 15 which is newer. Their version numbers are terrible

Reply:
yeah had trouble figuring that out. I did add opensuse-leap 15 and been a while since I've contributed to sous-chefs, but believe support is for latest and 1 previous version of OS. That's the only reason why I put 42 in.

Reply:
removed in latest commit.